pyproject.toml

```diff
@@ -20,17 +20,18 @@ classifiers = [
     "Operating System :: OS Independent",
 ]
 dependencies = [
-	"argparse",
-	"requests",
-	"PyYAML",
-	"dictdiffer"
+    "argparse",
+    "argcomplete",
+    "requests",
+    "PyYAML",
+    "dictdiffer"
 ]
 
 [project.optional-dependencies]
 tests = [
-  "pytest",
-  "pytest-cov",
-  "requests-mock",
+    "pytest",
+    "pytest-cov",
+    "requests-mock",
```

Comment:
Is this change intentional?

Reply:
Yes, I wanted to have the same indentation everywhere in pyproject.toml